recipe/conda_build_config.yaml

```diff
@@ -18,20 +16,21 @@ cudatoolkit:
 cudnn:
   - 7.6
 blas_impl:
-  - mkl                        # [x86 or x86_64]
-openblas:
-  - 0.3.3
+  - mkl         # [x86]
+  - openblas    # [not x86]
 mkl:
-  - 2019
+  - 2020        # [linux]
+  - 2019        # [osx]
+openblas:
+  - 0.3.10
 numpy:
-  - 1.11
+  - 1.16
 MACOSX_DEPLOYMENT_TARGET:
   - 10.10
```

Comment:
this needs adjustment to distinguish between osx-64 and osx-arm64.  I assume this block should be made used only for osx-64

Reply:
This comment is based on an older version of the PR and is not applicable anymore.